src/middlewares/findUsers.js

```diff
@@ -9,4 +10,24 @@ const findOneUser = async (req, res, next) => {
   }
   next();
 };
+export const commentAccess = async (req, res, next) => {
+  const { id } = req.user;
+  const { tripRequestId } = req.params;
+  const isManager = await users.findOne({
+    where: {
+      id,
+      roleId: 6,
+    }
+  });
+  const isRequester = await tripRequests.findOne({
+    where: {
+      id: tripRequestId,
+      userId: id,
+    }
+  });
```

Comment:
Same here! database queries can be put elsewhere. I tend to think this will help us have modular code.